sql_generators/glean_usage/templates/usage_reporting_clients_daily_v1.query.sql

```diff
@@ -0,0 +1,108 @@
+{{ header }}
+
+WITH usage_reporting_base AS (
+  SELECT
+    submission_timestamp,
+    DATE(submission_timestamp) AS submission_date,
+    metrics.uuid.usage_profile_id,
+    normalized_channel,
+    -- client_info.app_display_version,
+    -- client_info.app_build,
+    normalized_os,
+    normalized_os_version,
+    -- client_info.locale,
+    normalized_country_code,
```

Comment:
@irrationalagent Are we ok to include the country code instead of `locale` (no longer available due to it coming from the client_info)?

Reply:
country code should stay, this is not reported in the ping itself but assigned on the backend